spec/unit/migrations/helpers_spec.rb

```diff
@@ -0,0 +1,68 @@
+describe Neo4j::Migrations::Helpers do
+  include described_class
+
+  before do
+    clear_model_memory_caches
+    delete_db
+
+    stub_active_node_class('Book') do
+      property :name, constraint: :unique
+      property :author_name, index: :exact
+    end
+
+    Book.create!(name: 'Book1')
+    Book.create!(name: 'Book2')
+    Book.create!(name: 'Book3')
+  end
+
+  describe '#rename_property' do
+    it 'renames a property' do
+      rename_property :Book, :name, :title
+      expect(Book.all(:n).pluck('n.title')).to include('Book1', 'Book2', 'Book3')
```

Comment:
We should test whatever edge-case behavior we decide on regarding my comment above about what happens to nodes which already have the new property set.